tensorflow/java/BUILD

```diff
@@ -111,6 +111,11 @@ cc_binary(
             LINKER_EXPORTED_SYMBOLS,
         ],
         "//tensorflow:windows": [],
+        "//tensorflow:debug" : [
```

Comment:
You might want to put these into a separate select block and + it with this one. This existing select block appears to be used for operating system detection.